LocalyticsXamarin/Android/MainActivity.cs

```diff
@@ -25,6 +25,10 @@ protected override void OnCreate(Bundle bundle)
         {
             base.OnCreate(bundle);
 
+            var refreshedToken = FirebaseInstanceId.Instance.Token;
```

Comment:
Not sure how this translated to Xamarin, but this method call has also been updated to a callback style function: https://github.com/localytics/Android-Client-Library/blob/develop/android-client-library/src/main/java/com/localytics/android/AnalyticsHandler.java#L2110-L2123

Reply:
Same with the above comment. Don't need it, just messing around. I'll fix this one though so we can have a simple "firebase connected" textbox on the sample app